docs/source/roadmaps/may-2024.md

```diff
@@ -0,0 +1,143 @@
+# May 2024
+
+One of our main goals in the team is to provide fully integrated, automated analysis tools for systems neuroscience 
+data. As a researcher, this would mean providing only a set of configurations before running the analysis for an entire 
+dataset, with results and quality metrics output for checking. This will avoid researchers needing to write 
+custom preprocessing and analysis scripts that often duplicate existing efforts, and allows researchers to work 
+together on community tools.
+
+This is a long-term goal, and we are working across multiple themes towards achieving this. We are ensuring [data is 
+organised in a standard way](data_management), a critical step towards automation. Currently, we are focused on 
+building pipelines for [behaviour](behaviour), [electrophysiology](electrophysiology) and [anatomy](anatomy) data that 
+can ingest data from various acquisition systems into a common interface. These are critical steps towards the 
+longer-term goal of multimodal integration. The detailed roadmap is split into these themes below.
+
+(data_management)=
+## Data management
+**Current status**
+
+We have developed [NeuroBlueprint](https://neuroblueprint.neuroinformatics.dev/), a simple, standard data folder 
+specification for (systems) neuroscience and [datashuttle](datashuttle.neuroinformatics.dev), to automate the 
+creation, validation and transfer of NeuroBlueprint projects. For more details on this specific project, see the 
+[datashuttle roadmap]().
+
+**Q4 2024**
+
+Standardised metadata support
+
+**Q2 2025**
+
+Support for customising data structure (e.g. specific mandatory elements or new file types)
+
+**Q4 2025**
+
+Export of NeuroBlueprint projects to [Neurodata Without Borders](https://www.nwb.org/) format.
+
+(behaviour)=
+## Behavioural analysis
+**Current status**
+
+We are building [movement](https://movement.neuroinformatics.dev/) a Python toolbox for analysis of animal tracking 
+data. For more details on this specific project, see the [movement roadmap](https://movement.neuroinformatics.dev/community/roadmap.html).
+
+**Q3 2025**
+
+- Support all common post pose-estimation analysis tasks
+- Support for single experiments and complex multi-animal, multi-session datasets 
+(by reading and writing to [NeuroBlueprint](https://neuroblueprint.neuroinformatics.dev/) format)
+
+**No current ETA**
+
+- A common interface to prevalent pose estimation tools, and other machine-learning based object detection methods
+- A common interface to methods for behavioural segmentation and classification
```

Comment:
```suggestion
- A common interface to methods for behavioural segmentation and classification
```
I like that the first point is included with "No current ETA" (it would be very useful and we have chatted a lot about it) but I think the second one is still too broad/ambiguous in our minds, so maybe worth removing.

Reply:
I'm keen to keep this in, just because it's one of the things I've been asked for that, as you say, we don't have a plan to tackle. We may get feedback that would help us focus our ideas a bit.